src/olympia/addons/migrations/0011_auto_20200609_0522.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.13 on 2020-06-09 05:22
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('addons', '0010_auto_20200604_1319'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='addon',
+            name='view_source',
+        ),
```

Comment:
I have a question.
When I made migrations file, `public_stats` filed is occurred that is Remove Field.
But I deleted line about `public_stats`.
Should I include `public_stats` property?